zebra-consensus/src/block/subsidy.rs

```diff
@@ -140,22 +146,20 @@ mod test {
         let blossom_height = Blossom.activation_height(network).unwrap();
         let canopy_height = Canopy.activation_height(network).unwrap();
 
-        assert_eq!(
-            Amount::try_from(250_000_000),
-            founders_reward(SLOW_START_INTERVAL, network)
-        );
+        // Founders reward is 20% of the block subsidy
+        // Before Blossom this is 20*12.5/100 = 2.5 ZEC
         assert_eq!(
             Amount::try_from(250_000_000),
             founders_reward((blossom_height - 1).unwrap(), network)
         );
+        // Founders reward is still 20% of the block subsidy but the block reard is half what it was
```

Comment:
Typo
```suggestion
        // Founders reward is still 20% of the block subsidy but the block reward is half what it was
```

Reply:
This is fixed already in https://github.com/ZcashFoundation/zebra/pull/1051/commits/750355f625aada6e170e3f270baede680c0b2a56